dev/com.ibm.ws.security.social/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -238,3 +241,19 @@ discoveryPollingRate.desc=Rate in milliseconds at which the OpenID Connect clien
 
 forwardLoginParameter=Forward login parameter
 forwardLoginParameter.desc=Specifies a comma-separated list of parameter names to forward to the OpenID Connect provider. If a protected resource request includes one or more of the specified parameters, the OpenID Connect client will include those parameters and their values in the authorization endpoint request to the OpenID Connect provider.
+
+serviceAccountToken=Service account token
+serviceAccountToken.desc=Token for OpenShift service account with the system:auth-delegator role. The token is required for making requests to the TokenReview API.
+
+oauthServer=OpenShift OAuth server host and post
+oauthServer.desc=Host name and port number of the OpenShift server.
+
+useAccessTokenFromRequest=Use access token from request
+useAccessTokenFromRequest.desc=Determines if the access token that is provided in the request is used for authentication.
+
+useAccessTokenFromRequest.no=Do not use the access token that is provided in the request for authentication.
+useAccessTokenFromRequest.required=The access token that is provided in the request must be used for authentication.
+useAccessTokenFromRequest.ifPresent=Use the access token that is provided in the request for authentication if it is present.
```

Comment:
Change
"Use the access token that is provided in the request for authentication if it is present."
to
"Use the access token that is provided in the authentication request if it is present."